packages/main/test/specs/MultiInput.spec.js

```diff
@@ -91,26 +91,31 @@ describe("MultiInput general interaction", () => {
 	});
 
 	it ("adds an overflowing token to multi input", async () => {
+
+
 		const mi = await browser.$("#multiple-token");
 		const btn = await browser.$("#add-to-multiple");
 
 		assert.strictEqual(await mi.$$("ui5-token").length, 5, "should have 5 token");
 		$("#suggestion-token").scrollIntoView();
 
 		let allTokens = await mi.$$("ui5-token");
-		assert.notOk(await allTokens[0].getProperty("overflows"), "Token should not overflow");
 
-		for (let i = 1; i <= 4; i++) {
-			assert.ok(await allTokens[i].getProperty("overflows"), "Token should overflow");
+		assert.notOk(await allTokens[0].getProperty("overflows"), `Token 0 should not overflow`);
+		assert.notOk(await allTokens[1].getProperty("overflows"), `Token 1 should not overflow`);
+		for (let i = 2; i <= 4; i++) {
+			assert.ok(await allTokens[i].getProperty("overflows"), `Token ${i} should overflow`);
```

Comment:
For me this loop is unnecessary you could just test the 2 tokens and it would be 2 lines instead of 3